cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/graph/edges/flows/DataflowTest.kt

```diff
@@ -101,4 +117,35 @@ class DataflowTest {
             assertEquals(0, node2.prevDFGEdges.size)
         }
     }
+
+    @Test
+    fun testThrow() {
+        val result =
+            TestLanguageFrontend().build {
+                translationResult {
+                    translationUnit("some.file") {
+                        function("foo", t("void")) {
+                            body {
+                                declare { variable("a", t("short")) { literal(42) } }
+                                `throw` { call("SomeError") { /* TODO parameter a */} }
+                            }
+                        }
+                    }
+                }
+            }
+
+        // Let's assert that we did this correctly
+        val main = result.functions[0] // TODO: why can't I use "foo" like with the other tests?
+        assertNotNull(main)
+        val body = main.body
+        assertIs<Block>(body)
+
+        val throwStmt = body.statements.getOrNull(0)
```

Comment:
```suggestion
        val throwStmt = body.statements.getOrNull(1)
```
Statement 0 is the declaration statement of `a`.